requirements.txt

```diff
@@ -0,0 +1,25 @@
+bok-choy==1.1.1
+
+Django>=1.11,<2; python_version < '3'
+Django>=2.2,<2.3; python_version >= '3'
+
+django-pyfs==2.2
+flake8==3.8.4
+mock==3.0.5
+
+pytest==4.6.11; python_version < '3'
+pytest==6.1.2; python_version >= '3'
+
+pytest-cov==2.10.1
+
+pytest-django==3.10.0; python_version < '3'
+pytest-django==4.1.0; python_version >= '3'
+
+six==1.15.0
+
+web-fragments==0.3.2
+
+XBlock==1.2.1; python_version < '3'
+XBlock==1.4.0; python_version >= '3'
+
+xblock-sdk==0.2.2
```

Comment:
Stylistically, I'm not a big fan of locking versions. I'm willing to defer to your lead here, but my preference is >= rather than == for Python 3 versions wherever possible.

Rationale: 

* For development/sandbox/non-deployment settings, I prefer to get the latest version, and fix bugs when and if they occur. Otherwise, you end up with piling debt, and major ports. If you develop with a range of versions, code tends to run up fewer version dependencies. 

* For deployment, I like stable configurations, but I assume versions will come from the install of edX platform. Indeed, locking versions can make a mess if one release of edX wants 1.15 of something, another wants 1.13, and the XBlock is at a loss for which one to lock to.

Reply:
I like this approach. I can actually go with unlocked versions and it won't be bad, except for Python 2.7 compatibility which I like to preserve for a while.